### PR TITLE
AB#39437 string updates

### PR DESCRIPTION
**Pull request by acchiang**
- [x] Rename "Connected Resource" to "Resource IDs"
- [x] Include Team ID when there is a Channel ID
- [x] Error message should say "Please install to a Teams channel or chat instance"
- [x] Config screen should include the string: :"By installing this app, you are granting it resource-specific consent permission against the Teams channel/chat"

### CODE REVIEW

File: tabs/src/components/TabConfig.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ class TabConfig extends React.Component {
     return (
       <center className = "center" >
         <img className = "image" src={imageName.default} alt = ''/>
-        <h4 className = "caption">Select save to finish adding the Graph Explorer Sample Teams app.</h4>
+        <h4 className = "caption">Select save to finish adding the Graph Explorer Sample Teams app. By installing this app, you are granting it resource-specific consent permission against the Teams channel/chat. </h4>
```
Comment (ayubun) [inline tabs/src/components/TabConfig.jsx]: Is there a reason this isn’t a localized variable? 👀

**Review by ayubun — CHANGES_REQUESTED** [review]
There's actually a few places that need a re-check on this PR @acchiang since I just merged #15 and so the localization framework is in place / should now be upkept in each new PR if possible 🙏 If you have too much on your plate I don't mind helping out by snagging the responsibility, but I wanted to at least let you know first / leave the option to ya!

The rest of the PR looks good and localization looks good for there too c: LGTM after localization re-review !

File: tabs/src/components/TabConfig.jsx (hunk 1 above)
Comment (acchiang) [reply tabs/src/components/TabConfig.jsx]: Nope I just added the string outlined in the ticket and followed the file's standards so far, didn't know this should be localiized! I'll do that

**Comment by acchiang** [comment]
Thanks for the comments @AyuAi! It's ready for re-review :)

**Review by lhao03 — COMMENTED** [review]
i got an error about using react hooks in class components, so i changed TabConfig to a function component in this branch https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/tree/interns/lhao03/39437-change-to-functional-compnent and it seems to be working.

image


also are these two ids supposed to be slightly different colors?
image

**Comment by anneguo3** [comment]
Small copy change request - could we do just "Team ID" and "Channel ID"?

**Comment by lhao03** [comment]
@acchiang i made a PR for the functional component, you just need to give it a review! https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/46

**Review by lhao03 — APPROVED** [review]
lgtm!! (～￣▽￣)～

**Review by ayubun — APPROVED** [review]
Berry noice c: LGTM!

**Pull request merged**